linkerd/buffer/Cargo.toml

```
@@ -8,12 +8,13 @@ publish = false
 [dependencies]
 futures = "0.3"
 linkerd2-error = { path = "../error" }
-tokio = { version = "0.2", features = ["sync", "stream", "macros"] }
+tokio = { version = "0.2", features = ["sync", "stream", "time", "macros"] }
 tower = "0.3"
 tracing = "0.1"
 tracing-futures = { version = "0.1", features = ["std-future"] }
 pin-project = "0.4"
 
 [dev-dependencies]
+tower = { version = "0.3", features = ["util"] }
```

nit: let's also add
```suggestion
tower = { version = "0.3", default_features = false, features = ["util"] }
```
to tower dependencies.

We don't need that to be enabled, since we are collecting all tracing events as traces already.